# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sysvinit/dev-initctl.patch
# Copyright (C) 2018 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The Debian folks changed the initctl from /dev to /run which does not
jsut work, also not mounted write-able early enough, and, .. bäh, ..!

  - René Rebe <rene@exactcode.de>

--- sysvinit-2.90/src/initreq.h.vanilla	2018-08-19 14:16:26.533756646 +0000
+++ sysvinit-2.90/src/initreq.h	2018-08-19 14:17:04.353756078 +0000
@@ -26,7 +26,7 @@
 #include <sys/param.h>
 
 #ifndef INIT_FIFO
-#define INIT_FIFO  "/run/initctl"
+#define INIT_FIFO  "/dev/initctl"
 #endif
 
 #define INIT_MAGIC 0x03091969
